scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -125,44 +125,59 @@ function verify_subm_op_pod() {
   # TODO: Verify logs?
 }
 
+function validate_pod_container_equals() {
+  validate_equals ".spec.containers[].${1}" $2
+}
+
+function validate_pod_container_has() {
+  local json_filter=".spec.containers[].${1}"
+  local expected=$2
+  [[ $(jq -r -M $json_filter $json_file) =~ $expected ]]
+}
+
+function validate_pod_container_env() {
+  local var_name=$1
+  local expected=$2
+  [[ $(jq -r -M ".spec.containers[].env[] | select(.name==\"${var_name}\").value" $json_file) = $expected ]]
```

Comment:
In `kubectl jsonpath` this can be written `.spec.containers.env[?(@.name="${var_name}")].value`, not sure whether that would work here.

Reply:
Doesn't seem like they use the same underlying library

```
[root@04239f4bedaa submariner-operator]# jq '.spec.containers[].env[?(@.name="SUBMARINER_NAMESPACE")].value' //tmp/pod.json
jq: error: syntax error, unexpected '?' (Unix shell quoting issues?) at <top-level>, line 1:
.spec.containers[].env[?(@.name="SUBMARINER_NAMESPACE")].value                       
```